src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java

```diff
@@ -89,7 +89,9 @@ public CallbackResponse reclassifyExceptionRecord(
         @RequestBody CcdCallbackRequest callbackRequest,
         @RequestHeader(value = USER_ID, required = false) String userId
     ) {
-        if (callbackRequest.getCaseDetails() == null) {
+        if (userId != null && userId.matches("^.*[\\n|\\r|\\t]+.*$")) {
```

Comment:
is this the only controller that should perform this validation?

Reply:
Ideally, there should be more, but this one definitely needs that, as this user ID is logged. It's a quick fix. We may add this validation to other endpoints later.

Reply:
do we use any validation api ?

Reply:
we use vavr + custom validator for validating exception record details